### PR TITLE
i18n(es): update auto.mdx

### PR DESCRIPTION
**Pull request by AitorMT**
#### Description (required)

Updated the "**next steps**" section with the `CardGrid` and `LinkCard` components.

### CODE REVIEW

File: src/content/docs/es/install/auto.mdx [hunk 1]
```diff
@@ -132,8 +132,22 @@ Explora nuestros [temas y proyectos de inicio](https://astro.build/themes/) dond
...
-📚 **Añade un framework:** Aprende cómo extender Astro con soporte para React, Svelte, Tailwind y más, usando `npx astro add` en nuestra [Guía de integraciones](/es/guides/integrations-guide/).
-
-📚 **Despliega tu sitio:** Aprende cómo hacer el build y desplegar tu proyecto de Astro en la web con nuestra [Guía de despliegue](/es/guides/deploy/).
-
-📚 **Entiende el código base:** Aprende más sobre la estructura de proyectos de Astro en nuestra [Guía de estructura de proyectos](/es/core-concepts/project-structure/).
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+<CardGrid>
+  <LinkCard
+    title="Entiende el código base"
```
Comment (Waxer59) [inline src/content/docs/es/install/auto.mdx:139]:
```suggestion
    title="Entiende tu base de código"
```

File: src/content/docs/es/install/auto.mdx [hunk 2]
```diff
@@ -132,8 +132,22 @@ Explora nuestros [temas y proyectos de inicio](https://astro.build/themes/) dond
...
+    description="Aprende más sobre la estructura de proyectos de Astro en nuestra Guía de estructura de proyectos."
```
Comment (Waxer59) [inline src/content/docs/es/install/auto.mdx:140]:
```suggestion
    description="Aprende más sobre la estructura de archivos de Astro en nuestra Guía de Estructura del Proyecto."
```

File: src/content/docs/es/install/auto.mdx [hunk 3]
```diff
@@ -132,8 +132,22 @@ Explora nuestros [temas y proyectos de inicio](https://astro.build/themes/) dond
...
+    href="/es/core-concepts/project-structure/"
+  />
+  <LinkCard
+    title="Añade un framework"
+    description="Aprende cómo extender Astro con soporte para React, Svelte, Tailwind y más, usando npx astro add en nuestra Guía de integraciones"
+    href="/es/guides/integrations-guide/"
+  />
+  <LinkCard
+    title="Despliega tu sitio"
+    description="Aprende cómo hacer el build y desplegar tu proyecto de Astro en la web con nuestra Guía de despliegue."
```
Comment (Waxer59) [inline src/content/docs/es/install/auto.mdx:150]:
```suggestion
    description="Aprende cómo construir y desplegar tu proyecto de Astro en la web con nuestra Guía de despliegue."
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🚀

**Pull request merged**